src/users/containers/users/users.component.ts

```diff
@@ -32,7 +28,6 @@ export class UsersComponent implements OnInit, OnDestroy {
       { header: 'status', key: 'status' }
     ];
 
-
     this.store.dispatch(new fromStore.LoadUsers());
     this.userSubscription = this.store.pipe(select(fromStore.getGetUserArray)).subscribe(userData => {
```

Comment:
Why are yoou re-mapping data. Why yo don't put the observable directly in the html

(userSubscription | async)

and have it clean as 'this.userSubscription = this.store.pipe(select(fromStore.getGetUserArray))'

Reply:
you probably right... i haven't worked on this just as yet.. this pr is only about invite users... this will be next in pipeline